File: develop/lab-3-dynamodb/dynamoDBJavaLab/src/test/java/com/amazonaws/lab/ReservationsStatisticsTest.java
package com.amazonaws.lab;
// Copyright 2017 Amazon Web Services, Inc. or its affiliates. All rights reserved.

import static org.junit.Assert.*;

import org.junit.Test;

public class ReservationsStatisticsTest {

	@Test
	public void test() throws Exception {
		ReservationsStatistics.main(new String[] { "Reno" });
		assertEquals(178, ReservationsStatistics.itemCount, 0);
	}
}

File: develop/lab-3-dynamodb/dynamoDBPythonLab/reservations_statistics_test.py
# Copyright 2017 Amazon Web Services, Inc. or its affiliates. All rights
# reserved.

import unittest
import reservations_statistics
import warnings


class ReservationsStatisticsTest(unittest.TestCase):

    def test_reservations_statistics(self):
        warnings.simplefilter("ignore", ResourceWarning)
        itemCount = reservations_statistics.query_by_city("Reno")
        self.assertEqual(178, itemCount)

if __name__ == '__main__':
    unittest.main()
